services/wallet/service.go

```diff
@@ -426,3 +431,7 @@ func (s *Service) GetCollectiblesService() *collectibles.Service {
 func (s *Service) GetCollectiblesManager() *collectibles.Manager {
 	return s.collectiblesManager
 }
+
+func (s *Service) LocalBackup() *localbackup.BackupManager {
```

Comment:
Ideally, the wallet service/module shouldn't be aware of the term `localbackup`. It should only provide a way to serialize/deserialize itself. How this functionality is used, i.e whether by localbackup or something else, should be the responsibility of the wallet service/module's client.

Reply:
This is the aftermath of `BackupProvider` interface I suppose :disappointed:

Reply:
I agree that `LocalBackup` is not good to see here. That's why I'd not extract this to a separate `localbackup.BackupManager`, rather implement required methods of `BackupProvider` directly in the Service.

But having the word `Backup` in the service IMO is fine. In terms of `status-go` the goal of service backup is clear. Comparing to vague "serialize/deserialize", which is usually used for messages. We never send the service anywhere 😄